apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-steps.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import type { FormEvent } from 'react';
```

Comment:
comment: cc @elmar-vega @macqbat, this may make adding fraction orders easier to your application 👍